### ISSUE TITLE
Push Notification

### ISSUE THREAD

**Issue by kripsbro** [metrue/Applize#1]
Notifications.js won't work :( can you fix them ?

**Comment by metrue** [comment]
@kripsbro 

Yeah, I will.  but if you give me more details about this issue will help a lot.

**Comment by kripsbro** [comment]
So i try to open https://web.skype.com and set UserAgent to UIWebView   'NSString\* userAgent = @"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_11_3) AppleWebKit/601.4.4 (KHTML, like Gecko) Version/9.0.3 Safari/601.4.4";' when i receive Notifications in Skype i don't see them as alert :(

**Comment by metrue** [comment]
@kripsbro  

I will take a look at it this weekend.

**Comment by kripsbro** [comment]
Thanks, can you send a dome of pushnotification and website witch work ?

**Comment by metrue** [comment]
@kripsbro 

What I want to clarify is that this <code> notification </code> is not this iOS [notifications](https://developer.apple.com/notifications/).

What I do here is binding web browser's web notification [Notification](https://developer.mozilla.org/en-US/docs/Web/API/Notifications_API/Using_the_Notifications_API) to iOS [alert](https://developer.apple.com/library/ios/documentation/UserExperience/Conceptual/UIKitUICatalog/UIAlertView.html). 

So basically you will see, your website pops a alert means there is a notification even happens on your web website.

**Comment by kripsbro** [comment]
Yes, i know, so  web.skype.com sends WebNotifications, but  it wont receive
it :(

**Comment by metrue** [comment]
@kripsbro 

Oh, I see,  Let debug a little bit this weekend.

**Comment by kripsbro** [comment]
ok, thanks

**Comment by kripsbro** [comment]
did you checked ?

**Comment by kripsbro** [comment]
Hello

**Comment by metrue** [comment]
@kripsbro 

Sorry for later response, I did checked it.

the root cause is iOS safari does not support 'Notification' currently,  you can refer to this doc.
[https://developer.apple.com/library/mac/documentation/NetworkingInternet/Conceptual/NotificationProgrammingGuideForWebsites/Introduction/Introduction.html](https://developer.apple.com/library/mac/documentation/NetworkingInternet/Conceptual/NotificationProgrammingGuideForWebsites/Introduction/Introduction.html]

I'a still working on it how to inject 'Notification' support it now, but not have a clear thought yet.

**Comment by kripsbro** [comment]
ok thanks

**Comment by metrue** [comment]
@kripsbro 

Notification issue should be fixed now, you can check.

And I already add a testing local testing website under Applize.Web,  You can follow the README to do the testing.

if everything goes well, please help close this issue,  Thanks for your feedback always.

**Comment by kripsbro** [comment]
Hi did you tried web.skype.com to test notifications ?

**Comment by kripsbro** [comment]
it don't works ;(
